docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -1,8 +1,166 @@
+
+Use the following command to assign a submitter for your EigenPod:
+```bash
+/cli assign-submitter --execNode $NODE_ETH --podAddress $EIGENPOD_ADDRESS --sender $EIGENPOD_OWNER_PK
+```
+
+Consider using a cold key for the EigenPod Owner role. This key should be stored securely and used infrequently. For cold keys, we recommend using hardware wallets (e.g., Ledger, HSMSs) or smart contract multisigs (e.g., Safe). 
+
+We strongly recommend using a seperate key for the Proof Submitter, which can be considered a hot key. The Proof Submitter is any other address approved to submit proofs on behalf of the EigenPod owner. This separation allows the EigenPod owner key to remain secure and cold. Hot keys, while less secure, can be managed with solutions like Vault (Hashicorp) or environment variables. It is crucial not to store any meaningful value in your hot keys as operational keys are considered less secure. 
+
+
+
+## Steps to Deposit (Restake) Validator Native Beacon Chain ETH
+
+### Part 1: Create EigenPod
+
+Call EigenPodManager.createPod() .  
+
+### Part 2: Configure validator(s) withdrawal credentials
+
+1. Configure the validator(s) credentials to point to the EigenPod address when the validator is created. Please see [Ethereum Launchpad](https://launchpad.ethereum.org/en/withdrawals#enabling-withdrawals) for more information. 
+    a. Optional: you may choose to set the FEE_RECIPIENT to your EigenPod address if you wish to Restake those fees.
+
+2. Wait for the validator(s) to become active on-chain. Please see https://beaconcha.in/ to follow your validator status.
+
+### Part 3: Link the Validator to the EigenPod via Proof Generation
+
+1. Run the `credentials` command via the [EigenPod Proofs Generation CLI](https://github.com/Layr-Labs/eigenpod-proofs-generation/tree/master/cli#proof-generation).
+    
+
+2. Invoke the `credentials` command with the `--sender $EIGENPOD_OWNER_PK` argument so that CLI will submit proofs and act onchain for you. This is the private key of the wallet that was used to create the EigenPod. Example here:
+```bash
+./cli credentials --execNode $NODE_ETH --beaconNode $NODE_BEACON --podAddress $EIGENPOD_ADDRESS --sender $EIGENPOD_OWNER_PK
+```
+
+
+
+
+### Part 4: Run a Checkpoint Proof
+
+Generate [checkpoint proof ](https://github.com/Layr-Labs/eigenpod-proofs-generation/tree/master/cli#checkpoint-proofs)via eigenpod-proofs-generation CLI in order to initiate and complete a checkpoint.
+
+Your validator ETH balance is now Restaked.
```

Comment:
This doesn't need to happen here -- once you finish step 3, your validator's ETH balance is restaked.

Running checkpoint proofs is only helpful once you start accumulating rewards, or want to exit your validator.

Reply:
I thought so too, but @jbrower95 told me step 4 was necessary!?

Can we have a [duel](https://www.youtube.com/watch?v=pGxyOeypYFs) to determine who is correct?